src/Stratis.Bitcoin.Features.Miner/Controllers/StakingRPCController.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Security;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using NBitcoin;
+using Stratis.Bitcoin.Controllers;
+using Stratis.Bitcoin.Features.Miner.Interfaces;
+using Stratis.Bitcoin.Features.Miner.Models;
+using Stratis.Bitcoin.Features.Wallet.Interfaces;
+using Stratis.Bitcoin.Utilities;
+
+namespace Stratis.Bitcoin.Features.Miner.Controllers
+{
+    /// <summary>
+    /// RPC controller for calls related to PoW mining and PoS minting.
```

Comment:
`/// RPC controller for calls related to PoS minting.`

Reply:
cool